CODEOWNERS

```diff
@@ -0,0 +1,48 @@
+# Data interface
+openmc/data/ @paulromano
+
+# Python bindings to C/C++ API
+openmc/lib/ @paulromano
+
+# Depletion
+openmc/deplete/ @drewejohnson
+tests/regression_tests/deplete/ @drewejohnson
+tests/unit_tests/test_deplete_*.py @drewejohnson
+
+# MG-related functionality
+openmc/mgxs_library.py @nelsonag
+src/mgxs.cpp @nelsonag
+src/mgxs_interface.cpp @nelsonag
+src/physics_mg.cpp @nelsonag
+src/scattdata.cpp @nelsonag
+src/xsdata.cpp @nelsonag
```

Comment:
Unless we can get @wbinventor back in the loop (!!) might as well add me for the MGXS PyAPI as well (I'll take the whole sub-directory).